client/components/payment-activity/payment-data-tile.tsx

```diff
@@ -53,9 +53,9 @@ const PaymentDataTile: React.FC< PaymentDataTileProps > = ( {
 	reportLink,
 } ) => {
 	return (
-		<div id={ id } className="wcpay-payment-data-highlights__item">
+		<div className="wcpay-payment-data-highlights__item">
 			<p className="wcpay-payment-data-highlights__item__label">
-				<span>{ label }</span>
+				<span id={ id }>{ label }</span>
```

Comment:
I don't understand why this `id` attribute needs to be moved but I don't see any CSS selector in `client/components/payment-activity/style.scss` that targets by id (prefixed with `#`), so I don't see this will affect anything.

Reply:
This is used for the `aria-labelledby` attribute for the value/amount.

E.g. to let screen-readers know that the TPV amount is labelled 'Total payment volume'.

I noticed this label wasn't being correctly parsed when I was writing a test to check the amount is using the correct currency symbol.